src/main/java/mate/academy/bookstoreapp/service/BookService.java

```diff
@@ -2,12 +2,13 @@
 
 import java.util.List;
 import mate.academy.bookstoreapp.dto.book.BookDto;
+import mate.academy.bookstoreapp.dto.book.BookDtoWithoutCategoryIds;
 import mate.academy.bookstoreapp.dto.book.BookSearchParametersDto;
 import mate.academy.bookstoreapp.dto.book.CreateBookRequestDto;
 import org.springframework.data.domain.Pageable;
 
 public interface BookService {
-    BookDto save(CreateBookRequestDto requestDto);
+    BookDto save(CreateBookRequestDto requestDto, String categoryName);

```

Comment:
add categorieIds list to requestDto
```suggestion
    BookDto save(CreateBookRequestDto requestDto);

```